js/ui/html_editor/modules/tableContextMenu.js

```diff
@@ -118,7 +135,7 @@ if(Quill) {
         }
 
         prepareCleanCallback() {
-            return () => { this._detachEvents(); };
+            return () => { this._detachEvents(); this._popupForm?.dispose(); };
```

Comment:
```suggestion
            return () => {
                this._detachEvents();
                this._popupForm?.dispose();
            };
```

It is worth keeping the code clear. You can skip the executable line when reading